public/static/js/projects.js

```diff
@@ -86,6 +86,9 @@ mciModule.controller('ProjectCtrl', function($scope, $window, $http, $location,
       resource_type: "TASK",
       label: "a previously passing test in a task fails",
       regex_selectors: taskRegexSelectors(),
+      extraFields: [
+        {text: "Test names matching regex", key: "test-regex", validator: null}
```

Comment:
Are empty regexes rejected?

Reply:
No, but I'm doing this validation only on the server